src/modules/audio/openal/Audio.h

```diff
@@ -50,9 +50,9 @@
 #include <OpenAL-Soft/al.h>
 #endif
 #else
-#include <AL/alc.h>
-#include <AL/al.h>
-#include <AL/alext.h>
+#include <alc.h>
+#include <al.h>
+#include <alext.h>
 #endif
```

Comment:
I wonder if the shortened include paths mean the ifdefs for macos/iOS can also be cleaned up, presumably they can now set the full include path in their project files.

Reply:
I think macOS frameworks (via the `-framework` parameter rather than `-l` or whatever) still require the framework name in the include path, and love's xcode project still uses frameworks.